blocks/init/src/blocks/components/scroll-to-target/docs/story.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import readme from './readme.md';
+import { ScrollToTargetEditor } from '../components/scroll-to-target-editor';
+import { ScrollToTargetOptions } from '../components/scroll-to-target-options';
+
+export default {
+  title: 'Components|Scroll To Target',
+  parameters: {
+    notes: readme,
+  },
+};
+
+const editorProps = {
+  scrollText: 'Scroll to target',
```

Comment:
These are not translatable?

Reply:
I don't think they should be, as these are only example strings for Storybook